git_calendar/build.py

```diff
@@ -85,6 +84,27 @@ def main(argv=sys.argv[1:]):
         shutil.copy(TEMPLATE_DIR/'style.css', Path(args.index).parent/'style.css')
         open(args.index, 'w').write(index)
 
+    if args.content:
+        timestamp = subprocess.check_output('date', encoding='utf8').strip() # subprocess to get timezone
+        git_hash = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"], encoding='utf8').strip()
+
+        env = jinja2.Environment(
+            loader=jinja2.FileSystemLoader([TEMPLATE_DIR]),
+            autoescape=jinja2.select_autoescape(['html', 'xml', '.j2.html',]),
+        )
+        env.filters['markdown'] = markdown_it.MarkdownIt().render
```

Comment:
We could probably move this whole block out into an `if args.index or args.content` and deduplicate it with the above.

Reply:
Thanks! I will de-duplicate. Working on it.